index.html

```diff
@@ -9,27 +9,27 @@
   <link rel="stylesheet" media="(min-width:451px)" href="tablet.css">
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 
 <body>
   <div class="section-1">
-    <header>
-      <div class="logo">
-        <a href="#" class="logo">My Logo</a>
-      </div>
-      <nav>
-        <div class="icon-div desktop-hidden">
-          <a href="javascript:void(0);" class="icon desktop-hidden">
-            <i class="fa fa-bars"></i>
-          </a>
+    <header class="menu-wrap">
+      <nav class="logo">
+        <a href="#" class="brand">David</a>
+        <input type="checkbox" class="toggler" id="toggle" onclick="toggle()"/>
+        <div class="hamburger">
+          <div></div>
         </div>
-        <div class="nav-links mobile-hidden">
-          <ol>
-            <li><a href="#" class="nav-link">Portfolio</a></li>
-            <li><a href="#" class="nav-link">About</a></li>
-            <li><a href="#" class="nav-link">Contact</a></li>
-          </ol>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li  onclick="remove()"><a href="#works" class="nav-links">Portfolio</a></li>
+                <li  onclick="remove()"><a href="#about" class="nav-links">About</a></li>
+                <li  onclick="remove()"><a href="#contact" class="nav-links">Contact</a></li>
```

Comment:
- It is not adisable to call JavaScripts methods in your HTML file, this might result to your page loading slower than normal, I suggest removing the `remove()` method and leave all coding functions in your JS file, please fix this 😄 
- Also, according to the project requirements, when a link is clicked, the mobile menu should disappear, this is not working as expected in your project. Please fix this 🙏